src/Symfony/Component/HttpKernel/Exception/ValidationException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Exception;
+
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+use Symfony\Component\Validator\Exception\ValidationFailedException;
+
+class ValidationException extends HttpException
+{
+    public function __construct(
+        private mixed $value,
+        private ConstraintViolationListInterface $violations,
+        int $statusCode = 422,
+        string $message = 'Validation failed',
+        ?\Throwable $previous = null,
+        array $headers = [],
+        int $code = 0,
+    ) {
+        $validationFailedException = new ValidationFailedException($value, $violations);
```

Comment:
What about adding previous to `ValidationFailedException` like:
```php
public function __construct(
    private mixed $value,
    private ConstraintViolationListInterface $violations,
    ?\Throwable $previous = null,
) {
```
And doing: 
```suggestion
        $validationFailedException = new ValidationFailedException($value, $violations, $previous);
```
So we always have a `ValidationFailedException` in `HttpException`.